radar/tools/atlas.md

```diff
@@ -0,0 +1,14 @@
+---
+title: "Atlas"
+ring: trial
```

Comment:
for the trial ring we should have successful use case in our organization (in that case we can add reference), otherwise I would move to asses